pandas/core/dtypes/common.py

```diff
@@ -120,17 +120,26 @@ def is_object_dtype(arr_or_dtype):
 
 
 def is_sparse(arr):
-    """
-    Check whether an array-like is a pandas sparse array.
+    """Check whether an array-like is a pandas sparse array.
+
+    Check that the one-dimensional array-like is a pandas sparse array.
+    Returns True if it is a pandas sparse array, not another type of
+    sparse array.
```

Comment:
It will take some research, but I think it would be useful to explain what are the use cases of this function, besides what it does.